Lib/test/test_class.py

```diff
@@ -740,6 +740,24 @@ class A(0, 1, 2, 3, 4, 5, 6, 7, **d): pass
         class A(0, *range(1, 8), **d, foo='bar'): pass
         self.assertEqual(A, (tuple(range(8)), {'foo': 'bar'}))
 
+    def testClassCallRecursionLimit(self):
```

Comment:
Test fails on Windows 32 bit. Possibly because MSVC's stack usage on unoptimized builds is enormous.

Reply:
We aren't consuming C stack, so that shouldn't matter.